packages/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -4675,7 +4683,9 @@ describe('validation logic', () => {
 
       describe('substring', () => {
         testErrorsAndWarnings('row var = substring("a", 5, 5)', []);
+        testErrorsAndWarnings('row var = substring("a", 5)', []);
```

Comment:
Similarly, our old definition for `substring` inaccurately required 3 args when 2 is fine.